src/scripts.js

```diff
@@ -95,6 +95,9 @@ function createRecipeModal(event) {
     </div>`;
     document.getElementById('saveBtn').addEventListener('click', toggleSaveRecipe);
     document.getElementById('printBtn').addEventListener('click', () => window.print());
+    document.getElementById('closeIcon').addEventListener('keypress', (event) => {
+      event.key === "Enter" ? collapseRecipe(event) : null
+    });
```

Comment:
Does this work? Since the modal isn't created until a user clicks a recipe I would think that this event listener wouldn't get assigned in time.

Reply:
Tom and I discovered that buttons have built in "enter" event listeners already. Maybe changing the role to "button" is adding that functionality and this event listener is extraneous?

Reply:
@qrispi - For 

> Does this work? Since the modal isn't created until a user clicks a recipe I would think that this event listener wouldn't get assigned in time.

This is _inside_ the `createRecipeModal()` function @qrispi. I can't get the fetch working properly because of the restructuring of the heroku app, but I think it should work.

@jsahim can you confirm?

Reply:
> Tom and I discovered that buttons have built in "enter" event listeners already. Maybe changing the role to "button" is adding that functionality and this event listener is extraneous?

Now _this_ I have no idea about lol. Will wait for Jeff to confirm

Reply:
So in reference to both the img role assignment of button and this event listener, I found that that - adding the role of button did not make the icon focusable, nor did it provide built in ENTER functionality. This is why I added the 1) tab index, which made it focusable and  2) the listener to make it enter-able. 

I did a little digging and on MDN - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

I found the following, which made me question what we can actually do with this button role:
_Note: If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and define event handlers for click and keydown events. This includes handling the Enter and Space keypresses in order to process all forms of user input. See the official WAI-ARIA example code._

Reply:
> So in reference to both the img role assignment of button and this event listener, I found that that - adding the role of button did not make the icon focusable, nor did it provide built in ENTER functionality. This is why I added the 1) tab index, which made it focusable and 2) the listener to make it enter-able.
> 
> I did a little digging and on MDN - https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
> 
> I found the following, which made me question what we can actually do with this button role: _Note: If using role="button" instead of the semantic __ or elements, you will need to make the element focusable and define event handlers for click and keydown events. This includes handling the Enter and Space keypresses in order to process all forms of user input. See the official WAI-ARIA example code._

So.....the button ARIA role is basically pointless except for screen readers? Interesting choice.

THANK YOU for this write up and research, Jeff!!!

Reply:
Ahh didn't realize assigning the event listener was in the createRecipeModal function! Thanks for clearing everything up Jeff! Good work!